### PR TITLE
linking terms+chrome extension to landing page and footer

### PR DESCRIPTION
**Pull request by tarafanlin**

### CODE REVIEW

File: components/core/NewWebsitePrototypeFooter.js [hunk 1]
```diff
@@ -105,52 +105,45 @@ export const NewWebsitePrototypeFooter = (props) => {
...
+            Download
+          </a>
+          <br />
+          <a css={STYLES_LINK} href="https://github.com/filecoin-project/slate/issues/126" target="_blank">
             Community
           </a>
           <br />
-          <a css={STYLES_LINK} href="/guidelines" target="_blank">
-            Guidelines
+          <a css={STYLES_LINK} href="/privacy">
```
Comment (jimmylee) [inline components/core/NewWebsitePrototypeFooter.js:140]: There is no `privacy` anymore, this is just `guidelines`

**Review by jimmylee — CHANGES_REQUESTED** [review]

File: pages/index.js [hunk 2]
```diff
@@ -824,6 +824,19 @@ export default class IndexPage extends React.Component {
                 lets you seamlessly upload files to your slates from anywhere on
                 the web.
               </h3>
+              <div css={STYLES_ACTIONS_RIGHT}>
+                <div
+                  css={STYLES_BUTTON_PRIMARY}
+                  style={{ marginTop: 16 }}
+                  onClick={() =>
+                    window.open(
+                      "https://chrome.google.com/webstore/detail/slate/gloembacbehhbfbkcfjmloikeeaebnoc?hl=en"
```
Comment (jimmylee) [inline pages/index.js:833]: Some extra whitelines here it looks like.

**Comment by jimmylee** [comment]
Thanks for doing this Tara!

File: pages/index.js (hunk 2 above)
Comment (tarafanlin) [reply pages/index.js:833]: I think this is just how prettier formats the code

File: pages/index.js [hunk 3]
```diff
@@ -620,7 +593,7 @@ export default class IndexPage extends React.Component {
       threshold: 0.1,
     };
 
-    const observer = new IntersectionObserver(function(entries, self) {
+    const observer = new IntersectionObserver(function (entries, self) {
```
Comment (jimmylee) [inline pages/index.js:596]: Extra whitespace here between anonymous `function` and `(entites`

File: pages/index.js (hunk 2 above)
Comment (jimmylee) [reply pages/index.js:771]: 👍  I need to get you a new computer :)

**Review by jimmylee — CHANGES_REQUESTED** [review]
One more and you're good!

File: pages/index.js (hunk 3 above)
Comment (tarafanlin) [reply pages/index.js:596]: this is also the prettier auto-formatting, I can't do anything about it if I have prettier setup

Comment (jimmylee) [reply pages/index.js:596]: Hmm that is strange....

**Comment by jimmylee** [comment]
LGTM, Will need to figure out whats happening here with your prettier at somepoint.

**Pull request merged**